src/main/java/lotto_auto/domain/MagicNumber.java

```diff
@@ -0,0 +1,62 @@
+package lotto_auto.domain;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class MagicNumber {
+    String magicNumber;
+    public List<List<Integer>> lottoTotalNumberList;
+    public static int threeMatches;
+    public static int fourMatches;
+    public static int fiveMatches;
+    public static int sixMatches;
+
+    public MagicNumber() {
+    }
+
+    public MagicNumber(String magicNumber, List<List<Integer>> lottoTotalNumberList) {
+        this.magicNumber = magicNumber;
+        this.lottoTotalNumberList = lottoTotalNumberList;
+    }
+
+    public void compareMagicNumber() {
+        List<Integer> magicNumberList = separateMagicNumber();
+
+        for (int j = 0; j < lottoTotalNumberList.size(); j++) {
+            matchesCnt(getLottoMatches(j, magicNumberList, 0));
+        }
+    }
+
+    private static void matchesCnt(int cnt) {
```

Comment:
```suggestion
    private void matchesCnt(int cnt) {
```

static일 필요는 없지 않을까요?